README.md

```diff
@@ -1,16 +1,3 @@
-> [!NOTE]  
```

Comment:
I think lines 1,2,3 and 9 should be kept. Line for those coming from 1.14.6

Reply:
Actually, matbe change the
`HDF5 2.0, scheduled for release in Fall 2025, will *only* support the CMake build system.`
to
`Starting with HDF5 2.0, *only* the CMake build system is supported.`